styles/designs/carnival/parts/_table-shapes.scss

```diff
@@ -245,3 +245,12 @@
     )
 ));
 
+@include create_shape('carnival:::AlternateShading', (
+    _components: (
+      map-merge($Table__Body, (
+        _components:(
+          $Table__Row--AlternateShading,
+        )
+      )),
+    )
+));
```

Comment:
Is `alternate-shading` a new class for tables? If yes, we need to make additional change in cookbook. We should add class `os-alternate-shading-container` to `os-table`. I think we can fix that in a separate issue. So I would leave this as is for now. Would you mind to add a comment above this shape, that it should be modified after cookbook change? Also can you add words `table` and  `shape` to the name, so it be more descriptive? (`AlternateShadingTableShape`).